WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -347,6 +351,10 @@ public static boolean supportsLayoutGridFeature(SiteModel site) {
         return site != null && (site.isWPCom() || site.isWPComAtomic());
     }
 
+    public static boolean supportsEmbedVariationFeature(SiteModel site, String minimalJetpackVersion) {
```

Comment:
Thanks @fluiddot `supportsStoriesFeature` & `supportsContactInfoFeature ` methods utilize the same logic as well so now looking at it I am wondering if this method should be even more generic. This is a helper class so we don't have to obsess over these specifics but sharing nonetheless :)

Reply:
Good point, I think current and future Jetpack blocks would determine their availability using the same info so makes sense to unify them in a single helper. When I began integrating the embed blocks in the main apps, I thought that each Jetpack block would have different specifications, but now I'm leaning towards the fact that only a few would require a specific helper. So instead of having one helper per block, we could have a generic helper and only have a helper for those that would require it 👍 .

Reply:
Agreed! 👍🏾

Reply:
Since this PR is already approved, I'm thinking to merge it as-it-is and apply the changes mentioned in previous comments in a new PR.

Reply:
Sounds good!